packages/dds/map/src/directory.ts

```diff
@@ -230,11 +230,13 @@ export interface IDirectoryDataObject {
 }
 
 export interface IDirectoryNewStorageFormat {
+    /** @deprecated - added to prevent buggy caching. remove once all loaders past 0.35 */
+    absolutePath?: string;
     blobs: string[];
     content: IDirectoryDataObject;
 }
 
-function serializeDirectory(root: SubDirectory, serializer: IFluidSerializer): ITree {
+function serializeDirectory(absolutePath: string, root: SubDirectory, serializer: IFluidSerializer): ITree {
```

Comment:
naming wise, these layers do not need to know the semantics of it (that it's absolute path), name of variables can state it's unique but stable over time identifier